web/src/components/ChangesView.res

```diff
@@ -57,8 +57,10 @@ module Pies = {
 
 @react.component
 let make = (~index: string) => {
+  let updateFilters = defaultUpdateFilters
```

Comment:
I think we need to create a rescript binding for the legacy FiltersForm, and use that instead.

Reply:
I tried a raw definition but then I was unable to use it via rescript.